### DISCUSSION TITLE
New line on X value or Treemap

### DISCUSSION THREAD

**Discussion by FlatFaceCat** [apexcharts/apexcharts.js#3837] [Q&A]
I would like to add a second row of text to the label of my treemap. 

I am trying to use the standard JS scripting for example: "Mon\nCat" 
to get: 
Mon
Cat

But instead am getting: Mon Cat

Is it possible with the treemap to have text on 2 rows?

Thanks

**Comment by mabolen** [comment]
"To enable multiline text and word wrapping, we have to provide an array of strings for each word we need to give a line break." - [from docs](https://apexcharts.com/docs/multiline-text-and-line-breaks-in-axes-labels/)

Works in Treemap for dataLabels if you provide a formatter function:

```
const splitLabel = (val, opts) => val.split(" ")

const options = {
	dataLabels: {
		formatter: splitLabel,
	},
}
```
